### PR TITLE
Build binaries for Linux on IBM Power pcc64le architecture

### PR DESCRIPTION
**Pull request by redmark-alt**
Similar to https://github.com/jaegertracing/jaeger/pull/1982

## Which problem is this PR solving?
- 

## Short description of the changes
-

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,9 +22,11 @@ ALL_SRC := $(shell find . -name '*.go' \
 # ALL_PKGS is used with 'go cover' and 'golint'
 ALL_PKGS := $(shell go list $(sort $(dir $(ALL_SRC))))
 UNAME := $(shell uname -m)
-#Race flag is not supported on s390x architecture
+#Race flag is not supported on s390x/ppc64le architecture
 ifeq ($(UNAME), s390x)
 	RACE=
+else ifeq ($(UNAME), ppc64le)
+  RACE=
```
Comment (yurishkuro) [inline Makefile]: please use the same whitespace as above/below

File: Makefile [hunk 2]
```diff
@@ -196,9 +196,9 @@ elasticsearch-mappings:
 build-examples:
 	esc -pkg frontend -o examples/hotrod/services/frontend/gen_assets.go  -prefix examples/hotrod/services/frontend/web_assets examples/hotrod/services/frontend/web_assets
 ifeq ($(GOARCH), s390x)
-	CGO_ENABLED=0 installsuffix=cgo go build -o ./examples/hotrod/hotrod-$(GOOS)-$(GOARCH) ./examples/hotrod/main.go
+	$(GOBUILD) -o ./examples/hotrod/hotrod-$(GOOS)-$(GOARCH) ./examples/hotrod/main.go
```
Comment (pavolloffay) [inline Makefile]: The build command for `s390x` and `ppc64le`  are looking the same. Could we merge them?

File: Makefile [hunk 3]
```diff
@@ -195,6 +197,8 @@ build-examples:
 	esc -pkg frontend -o examples/hotrod/services/frontend/gen_assets.go  -prefix examples/hotrod/services/frontend/web_assets examples/hotrod/services/frontend/web_assets
 ifeq ($(GOARCH), s390x)
 	$(GOBUILD) -o ./examples/hotrod/hotrod-$(GOOS)-$(GOARCH) ./examples/hotrod/main.go
+else ifeq ($(GOARCH), ppc64le)
```
Comment (pavolloffay) [inline Makefile]: The build command for s390x and ppc64le are looking the same. Could we merge them?

Comment (redmark-alt) [reply Makefile]: Changes made as requested.

**Comment by yurishkuro** [comment]
your branch seems messed up, I don't think you meant to update 150 files.

**Comment by redmark-alt** [comment]
Yes, I was attempting to process the DCO correction and the push went bad.  Let trash this branch and I'll start over.

**Pull request closed without merging**